elliptic-curve/src/ops.rs

```diff
@@ -23,6 +24,117 @@ pub trait Invert {
         // Fall back on constant-time implementation by default.
         self.invert()
     }
+
+    /// Perform a batched inversion on a sequence of field elements (i.e. base field elements or scalars)
+    /// at an amortized cost that should be practically as efficient as a single inversion.
+    /// This variation takes a const-generic array and thus does not require `alloc`.
+    fn invert_batch_array<const N: usize>(field_elements: &[Self; N]) -> CtOption<[Self; N]>
```

Comment:
Another name possibility to consider: `batch_invert_array` and `batch_invert_slice`.

I like that because "batch invert" is how I typically hear this process described.

Reply:
I was considering both too. I'll change. Anything besides that?

Reply:
Might have some feedback on `ToAffineBatch` but it seems mostly OK